codalab/lib/bundle_cli.py

```diff
@@ -2091,11 +2091,16 @@ def do_add_command(self, args):
 
         elif args.item_type == 'bundle':
             for bundle_spec in args.item_spec:
-                source_client, source_spec = self.parse_spec(bundle_spec)
+                if '//' in bundle_spec:
+                    source_worksheet_spec, source_bundle_spec = tuple(bundle_spec.split('//', 1))
```

Comment:
`tuple` is not needed, right?

Reply:
Yep, confused this with some other time where I needed a tuple around a split call. Removing